mathics/builtin/numbers/randomnumbers.py

```diff
@@ -735,22 +735,30 @@ class RandomSample(_RandomSelection):
     </dl>
 
     >> SeedRandom[42]
-    >> RandomSample[{a, b, c}]
-     = {a}
+    >> RandomSample[{a, b, c, d}]
+     = {b, d, a, c}
```

Comment:
We probably shouldn't hard-code an order here. The way to give an example but allow variation is using `...`: 

```
>> RandomSample[{a, b, c, d}]
     = {...}
```